src/richie/apps/core/templates/menu/language_menu.html

```diff
@@ -1,22 +1,29 @@
 {% load i18n menu_tags cms_tags %}{% spaceless %}
 {% get_current_language as current_language_code %}
 {% if languages|length > 1 %}
-    {% for code, name in languages %}
+  {% for code, name in languages %}
+    {% if forloop.first %}
+      <ul class="topbar__menu__list topbar__menu__list--aside">
```

Comment:
Can't you just put this inside the `if languages|length > 1`?

Reply:
But it is already no?

Reply:
But I mean the `{% if forloop.first %}` and last are not necessary right?

Reply:
You damn right!